migrations/scripts/2_deploy_board.js

```diff
@@ -0,0 +1,104 @@
+const realm = process.env.WITNET_EVM_REALM || "default"
+const addresses = require("../addresses")[realm]
+const settings = require("../settings")
+
+let WitnetProxy, WitnetRequestBoard
+
+module.exports = async function (deployer, network, accounts) {
+  try {
+    WitnetProxy = artifacts.require(settings.artifacts[realm].WitnetProxy || settings.artifacts.default.WitnetProxy)
+    WitnetRequestBoard = artifacts.require(
+      settings.artifacts[realm].WitnetRequestBoard || settings.artifacts[realm].WitnetRequestBoard
```

Comment:
This `||` falls into tautology.

```suggestion
      settings.artifacts[realm].WitnetRequestBoard || settings.artifacts[realm].WitnetRequestBoard
```

Reply:
For the same reason as stated before, the expression at the right should actually be `settings.artifacts.default.WitnetRequestBoard`